drivers/storage/scaleio/tests/config.yml

```diff
@@ -0,0 +1,40 @@
+rexray:
+  logLevel: debug
+  modules:
+    default-docker:
+      type: docker
+      libstorage:
+        service: scaleio
+      host: unix:///run/docker/plugins/docker.sock
+    mesos:
+      type: docker
+      libstorage:
+        service: scaleio
+      host: unix:///run/docker/plugins/mesos.sock
+      libstorage:
+        integration:
+          volume:
+            operations:
+              unmount:
+                ignoreUsedCount: true
+    rexray:
+      type: docker
+      libstorage:
+        service: scaleio
+      host: unix:///run/docker/plugins/rexray.sock
+libstorage:
+  service: scaleio
+  integration:
+    volume:
+      operations:
+        mount:
+          preempt: true
+scaleio:
+  endpoint: https://10.0.0.11/api
+  insecure: true
+  thinOrThick: ThinProvisioned
+  userName: admin
```

Comment:
Since scaleio is being propped up preconfigured... that is the preconfigured username and password. Those are static and are not changing.